### PR TITLE
Release Notes for July 17 2019

### PR DESCRIPTION
**Pull request by rhiananthony**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,109 @@
...
+* f1cebf6 Document --shared option for subscriptions script
+* 17a780f Add unsubscribe target to Makefile
+* 61dcd69 Support duplicate notification (#947)
+* a72bac1 Add test for duplicate notifications (#947)
+* 57e7f84 Refactor aggregation code slightly for clarity
+* 3a35b0a Hot Fix: Exclude API endpoints call from health check to reduce load
+* fb98182 Changed project_shortname to project_short_name (#191)
+* 16205cb Revert metadata field in index roll back (#965, #966)
+* e35dfc2 Remove excluded list and exclude whitelisted project (#1112)
+* 22cfc66 Generate `full metadata' TSV by optimizing on memory (#967, #968)
```
Comment (amarjandu) [inline RELEASE_NOTES.md]: formatting error

Comment (rhiananthony) [reply RELEASE_NOTES.md]: Updated - great catch :)
